test/libsolidity/syntaxTests/inlineAssembly/assignment_from_opcode_like.sol

```diff
@@ -7,4 +7,4 @@ contract C {
     }
 }
 // ----
-// ParserError 2314: (118-119): Expected '(' but got '}'
+// Warning 8261: (52-62): Variable is shadowed in inline assembly by an instruction of the same name
```

Comment:
Even though it is shadowing, why does the name resolution takes precedence over the builtin?